code/game/objects/items.dm

```diff
@@ -1555,3 +1565,38 @@
 /obj/item/update_atom_colour()
 	. = ..()
 	update_slot_icon()
+
+/// Modifies the fantasy variable
+/obj/item/proc/modify_fantasy_variable(variable_key, value, bonus, minimum = 0)
+	var/intended_target = value + bonus
+	value = max(minimum, intended_target)
+
+	var/difference = intended_target - value
+	var/modified_amount = bonus - difference
+	LAZYSET(fantasy_modifications, variable_key, modified_amount)
+	return value
+
+/// Returns the original fantasy variable value
+/obj/item/proc/reset_fantasy_variable(variable_key, current_value)
+	var/modification = LAZYACCESS(fantasy_modifications, variable_key)
+	if(!modification)
+		return current_value
+	return current_value - modification
+
+/obj/item/proc/apply_fantasy_bonuses(bonus)
+	SHOULD_CALL_PARENT(TRUE)
+	SEND_SIGNAL(src, COMSIG_ITEM_APPLY_FANTASY_BONUSES, bonus)
+	force = modify_fantasy_variable("force", force, bonus)
+	throwforce = modify_fantasy_variable("throwforce", throwforce, bonus)
+	wound_bonus = modify_fantasy_variable("wound_bonus", wound_bonus, bonus)
+	bare_wound_bonus = modify_fantasy_variable("bare_wound_bonus", bare_wound_bonus, bonus)
+	toolspeed = modify_fantasy_variable("toolspeed", toolspeed, -bonus/10, minimum = 0.1)
```

Comment:
Should these var keys be using `NAMEOF` so we can get compile time checking?

Same goes for all implementations of this and its brother proc

Reply:
Not necessarily, the name is just a unique identifier and not all of them use exact variable names.
There is a risk of someone creating a collision given that these names are somewhat generic, but nothing will break if the underlying variable names change

Reply:
I could potentially write something to watch out for collisions in case they do happen, e.g. throw a runtime if modify_fantasy_variable is called twice before reset_fantasy_variable is called.

Reply:
And then also write a unit test to make sure this can't happen